packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -248,6 +268,7 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
     return {
       privacy: "public",
       properties: {
+        ...(region ? { geo: region } : {}),
```

Comment:
Do we also need to set `rtmp_geo`? https://docs.daily.co/reference/rest-api/rooms/create-room#rtmp_geo